openfeature-provider/src/test/java/com/spotify/confidence/ConfidenceFeatureProviderTest.java

```diff
@@ -24,7 +24,7 @@ class ConfidenceFeatureProviderTest {
   public void setup() {
     fakeEngine = new FakeEventSenderEngine(new FakeClock());
     fakeFlagResolverClient = new ResolverClientTestUtils.FakeFlagResolverClient();
-    root = Confidence.create(fakeEngine, fakeFlagResolverClient);
+    root = Confidence.create(fakeEngine, fakeFlagResolverClient, "");
```

Comment:
Would read better if you pass `"clientKey"` instead of just `""`.

Reply:
For all these test places that is.

Reply:
Yeah I can do that 👍